microservices/workers/src/Worker/Rates.php

```diff
@@ -349,9 +349,13 @@ public function import(GearmanJob $serializedJob)
             $this->logger->debug('About to insert DestinationRates');
             $tpDestinationRateChunks = array_chunk($destinationRates, self::CHUNK_SIZE);
             foreach ($tpDestinationRateChunks as $destinationRates) {
-                $this
+                $affectedRows = $this
                     ->destinationRateRepository
                     ->insertIgnoreFromArray($destinationRates);
+
+                if ($affectedRows > 0) {
+                    $disableDestinations = false;
+                }
```

Comment:
What this logic actually means?

Reply:
Oh I see, it seems to Reload CGRateS destinations only if new prefixes where added